src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import 'utils/variables';
+@import 'blocks/page';
+@import 'blocks/header';
+@import 'blocks/cards.scss';
```

Comment:
The file extension shouldn't be specified when importing a SCSS file. Please remove the `.scss` extension from the import statement.